plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -123,18 +121,24 @@ class RayClusterConf:
 
     # For additional options, check:
     # https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml
-    auth: Dict[str, str] = default_field({"ssh_user": "ubuntu"})
+    auth: Dict[str, str] = field(default_factory=lambda: {"ssh_user": "ubuntu"})
```

Comment:
I want to understand why we are dropping type safety here for those things.
I thought we had typing for the stuff at https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml.

Reply:
I vaguely remember that we talked about this during one of our 1-1s :) . but yea,  `example-full`  doesn't actually show all the fields available to the users,  all the fields available are in the this (link)[https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml#L74]

I think we agreed that it doesn't make sense to add all the fields in the config.